templates/blog/_post_tags.html.twig

```diff
@@ -1,9 +1,9 @@
 {% if not post.tags.empty %}
     <p class="post-tags">
         {% for tag in post.tags %}
-            <span class="label label-default">
+            <a href="{{ path('blog_index', {'tag': tag.name}) }}" class="label label-default">
```

Comment:
@yceruto, What about to change a style of the current selected tag by adding a different CSS class for him (`label-primary` or `label-info`)? See https://getbootstrap.com/docs/3.3/components/#available-variations

Reply:
I like that, done.